awsdynamodb.json

```diff
@@ -0,0 +1,5829 @@
+                        "Table details fetched successfully"
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "custom",
+                "title": "Describe Table",
+                "view": "awsdynamodb_view.display_view"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "create global table",
+            "identifier": "create_global_table",
+            "description": "Create a global table from an existing table in the specified regions",
+            "verbose": "<ul><li><p>Conditions to create a global table : </p><ul><li>The table must have the same name as all of the other replicas</li><li>The table must have DynamoDB Streams enabled, with the stream containing both the new and the old images of the item.</li></ul></li><li><p>If you want to add a new replica table to a global table, each of the following conditions must be true:</p><ul><li>The table must have the same primary key as all of the other replicas.</li><li>The table must have the same name as all of the other replicas.</li><li>The table must have DynamoDB Streams enabled, with the stream containing both the new and the old images of the item.</li><li>None of the replica tables in the global table can contain any data.</li></ul></li><li><p>If global secondary indexes are specified, then the following conditions must also be met:</p><ul><li>The global secondary indexes must have the same name.</li><li>The global secondary indexes must have the same hash key and sort key (if present).</li></ul></li><li><p>If local secondary indexes are specified, then the following conditions must also be met:</p><ul><li>The local secondary indexes must have the same name.</li><li>The local secondary indexes must have the same hash key and sort key (if present).</li></ul></li></ul>",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "global_table_name": {
+                    "description": "Name of table to make global",
+                    "required": true,
+                    "data_type": "string",
+                    "order": 0,
+                    "contains": [
+                        "aws dynamodb table name"
+                    ],
+                    "primary": true
+                },
+                "replication_group": {
+                    "description": "Name of region to create global table(accepts comma separated regions)",
```

Comment:
```suggestion
                    "description": "Name of region to create global table (accepts comma separated regions)",
```

Reply:
Updated the code and resolved the issue.